ml_tools/eolearn/tests/test_train_split.py

```diff
@@ -36,56 +36,63 @@ def test_bad_args(bad_arg: Any, bad_kwargs: Any) -> None:
         TrainTestSplitTask(INPUT_FEATURE, OUTPUT_FEATURE, bad_arg, **bad_kwargs)
 
 
-def test_train_split() -> None:
-    shape = (1000, 1000, 3)
-    data = np.random.randint(10, size=shape, dtype=int)
+@pytest.fixture(name="eopatch")
+def eopatch_fixture() -> EOPatch:
+    eopatch = EOPatch()
+    eopatch[INPUT_FEATURE] = np.random.randint(10, size=(1000, 1000, 3), dtype=int)
+
+    return eopatch
 
-    indices = [(0, 2, 0, 2), (0, 2, 2, 4), (2, 4, 0, 2), (2, 4, 2, 4), (0, 4, 4, 8), (4, 8, 0, 4), (4, 8, 4, 8)]
-    for index, (i_1, i_2, j_1, j_2) in enumerate(indices, 1):
-        data[i_1:i_2, j_1:j_2, :] = index * 11
 
+@pytest.fixture(name="eopatch_pair")
+def eopatch_pair_fixture(eopatch: EOPatch) -> Tuple[EOPatch, EOPatch]:
     patch = EOPatch()
-    patch[INPUT_FEATURE] = data
+    patch[INPUT_FEATURE] = np.random.randint(10, size=(1000, 1000, 3), dtype=int)
+
+    return eopatch, patch
+
+
+def test_train_split(eopatch: EOPatch) -> None:
+    indices = [(0, 2, 0, 2), (0, 2, 2, 4), (2, 4, 0, 2), (2, 4, 2, 4), (0, 4, 4, 8), (4, 8, 0, 4), (4, 8, 4, 8)]
+    for index, (i_1, i_2, j_1, j_2) in enumerate(indices, 1):
+        eopatch[INPUT_FEATURE][i_1:i_2, j_1:j_2, :] = index * 11
 
     bins = [0.2, 0.5, 0.8]
     expected_unique = set(range(1, len(bins) + 2))
 
-    patch = TrainTestSplitTask(INPUT_FEATURE, OUTPUT_FEATURE, bins, split_type=TrainTestSplitType.PER_CLASS)(
-        patch, seed=1
+    eopatch = TrainTestSplitTask(INPUT_FEATURE, OUTPUT_FEATURE, bins, split_type=TrainTestSplitType.PER_CLASS)(
```

Comment:
create the task and call it in a separate line. You'll get rid of one line of code that way